src/components/contextual/pages/pool/PoolMigrationCard/PoolMigrationCard.vue

```diff
@@ -0,0 +1,125 @@
+<script setup lang="ts">
+import useNetwork from '@/composables/useNetwork';
+import { deprecatedDetails } from '@/composables/usePool';
+import NewPoolData from './NewPoolData.vue';
+import usePoolsQuery from '@/composables/queries/usePoolsQuery';
+import usePoolQuery from '@/composables/queries/usePoolQuery';
+import GradientCard from '@/components/cards/GradientCard/GradientCard.vue';
+
+/**
+ * TYPES
+ */
+type Props = {
+  poolId: string;
+};
+
+/**
+ * PROPS
+ */
+const props = defineProps<Props>();
+
+/**
+ * COMPOSABLES
+ */
+const { networkSlug } = useNetwork();
+
+const poolDeprecatedDetails = computed(() => deprecatedDetails(props.poolId));
+
+let newPoolQuery;
+if (poolDeprecatedDetails.value?.newPool) {
+  newPoolQuery = usePoolQuery(poolDeprecatedDetails.value?.newPool);
+}
+
+let suggestedPoolsQuery;
+if (poolDeprecatedDetails.value?.suggestedPools?.length) {
+  suggestedPoolsQuery = usePoolsQuery(
+    ref([]),
+    {},
+    {
+      poolIds: ref(poolDeprecatedDetails.value.suggestedPools),
+      first: 1000,
+    }
+  );
+}
+
+/**
+ * COMPUTED
+ */
+const loadingNewPool = computed(
+  () => newPoolQuery && newPoolQuery.isLoading.value
+);
+const newPool = computed(() => {
+  if (!newPoolQuery) {
+    return;
+  }
+  return newPoolQuery.data.value;
+});
+
+const loadingSuggestedPools = computed(
+  () => suggestedPoolsQuery && suggestedPoolsQuery.isLoading.value
+);
+const suggestedPools = computed(() => {
+  if (!suggestedPoolsQuery) {
+    return;
+  }
+  return suggestedPoolsQuery.data.value?.pages?.[0].pools;
+});
+</script>
+
+<template>
+  <BalLoadingBlock
+    v-if="loadingSuggestedPools || loadingNewPool"
+    class="mb-4 h-60 pool-actions-card"
+  />
+  <BalStack v-else vertical>
+    <GradientCard
+      :imgUrls="[
+        '/images/migration/migration-bg.jpg',
+        '/images/migration/migration-bg.avif',
+      ]"
```

Comment:
This shouldn't be defined here. Also the image shouldn't be called migration because it's likely we use it again for other features. I'd rename the images `/images/gradients/gradient-1` for now and put this logic in the gradient component.